custom_components/tuya_local/devices/goldair_gpcv340_heater.yaml

```diff
@@ -0,0 +1,62 @@
+name: Goldair Panel heater (GPCV340)
+products:
+  - id: ynjanlglr4qa6dxf
+    manufacturer: Goldair
+    model: NPH-2000G-WIFI
```

Comment:
Is this correct? It does not match the filename, but it does match the Kogan heater that this config is identical to.